examples/cov2d_simulation.py

```diff
@@ -78,7 +78,7 @@
 # To be consistent with the Matlab version in the numbers, we need to use the statements as below:
 logger.info('Generate random distributed rotation angles and obtain corresponding 2D clean images.')
 rots = qrand_rots(num_imgs, seed=0)
-imgs_clean = vol2img(vols[..., 0], rots)
+imgs_clean = vol2img(sim.vols[..., 0], rots)
```

Comment:
Shouldn't all this be done inside `Simulation`?

Reply:
As above, the reason to use this way is generating exact result as matlab code.

Reply:
As above, a notice has been created for issue #61.